x-pack/plugins/ml/public/application/components/ml_inference/components/processor_configuration.tsx

```diff
@@ -0,0 +1,348 @@
+                    'xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.advanced.fieldMapTitle',
+                    { defaultMessage: 'Fields' }
+                  )}
+                </h4>
+              </EuiTitle>
+              <EuiSpacer size="m" />
+              <EuiText color="subdued" size="s">
+                <p>
+                  <FormattedMessage
+                    id="xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.advanced.fieldMapDescriptionTwo"
+                    defaultMessage="The list of fields the model expects. {fieldsList}"
+                    values={{
+                      fieldsList: (
+                        <EuiPopover
+                          button={
+                            <EuiLink onClick={() => setIsPopoverOpen(!isPopoverOpen)}>
+                              See them here.
+                            </EuiLink>
+                          }
+                          isOpen={isPopoverOpen}
+                          closePopover={() => setIsPopoverOpen(false)}
+                          anchorPosition="downLeft"
+                        >
+                          <EuiCodeBlock isCopyable={true}>
+                            {JSON.stringify(modelInputFields, null, 2)}
+                          </EuiCodeBlock>
+                        </EuiPopover>
+                      ),
+                    }}
+                  />
+                </p>
+                <p>
+                  <FormattedMessage
+                    id="xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.advanced.fieldMapExtendedDescription"
+                    defaultMessage="If the fields for the incoming data differ, a {fieldMap} must be created to map the input document field name to the name of the field that the model expects. It must be in JSON format. {inferenceDocsLink}"
+                    values={{
+                      fieldMap: <EuiCode>{'field_map'}</EuiCode>,
+                      inferenceDocsLink: (
+                        <EuiLink external target="_blank" href={links.ingest.inference}>
+                          Learn more.
+                        </EuiLink>
+                      ),
+                    }}
+                  />
+                </p>
+              </EuiText>
+            </EuiFlexItem>
+            <EuiFlexItem grow={7}>
+              <EuiFormRow
+                fullWidth
+                labelAppend={
+                  <EuiFlexGroup gutterSize="xs" justifyContent="flexStart">
+                    <EuiFlexItem grow={false}>
+                      <EuiButtonEmpty
+                        iconType="pencil"
+                        size="xs"
+                        onClick={() => {
+                          setEditFieldMapping(!editFieldMapping);
+                        }}
+                      >
+                        {editFieldMapping
+                          ? CANCEL_EDIT_MESSAGE
```

Comment:
Maybe, you can edit "Clear" besides "Cancel" and "Save changes" in the Edit mode. If I click edit and get the exemplary text, I can save it, but I get an error message that the input field was not found in the input field name. I cannot continue. I need to know that I have to "Edit" and remove the exemplary text manually to be able to move forward.